src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1276,8 +1295,13 @@ def image_text_embedder(
             image_embeds.shape[-1],
         )
         image_embeds = image_embeds.reshape(new_size)
+        text_embeds = outputs.text_embeds
 
-        return (image_embeds, text_embeds)
+        # Last hidden states from text and vision transformers
+        text_model_last_hidden_states = outputs.text_model_output.last_hidden_state
+        vision_model_last_hidden_states = outputs.vision_model_output.last_hidden_state
+
+        return (text_embeds, image_embeds, text_model_last_hidden_states, vision_model_last_hidden_states)
```

Comment:
I'd make the variables singular to be consistent

```suggestion
        text_model_last_hidden_state = outputs.text_model_output.last_hidden_state
        vision_model_last_hidden_state = outputs.vision_model_output.last_hidden_state

        return (text_embeds, image_embeds, text_model_last_hidden_state, vision_model_last_hidden_state)
```